core/integration/file_test.go

```diff
@@ -192,3 +195,46 @@ func TestFileWithTimestamps(t *testing.T) {
 	require.Contains(t, ls, "Access: 1985-10-26 08:15:00.000000000 +0000")
 	require.Contains(t, ls, "Modify: 1985-10-26 08:15:00.000000000 +0000")
 }
+
+func TestFileContents(t *testing.T) {
+	c, ctx := connect(t)
+	defer c.Close()
+
+	src := c.Host().Directory(".")
+	alpine := c.Container().
+		From("alpine:3.16.2").
+		WithDirectory("/src", src).WithWorkdir("/src")
+
+	// Set three types of file sizes for test data,
+	// the third one uses a size larger than the max chunk size:
+	fileSizes := []int{
+		// core.MaxFileContentsChunkSize / 2,
+		// core.MaxFileContentsChunkSize,
+		core.MaxFileContentsChunkSize * 2,
+	}
+
+	wd, err := os.Getwd()
```

Comment:
I think `t.TempDir()` would be a better place, just so there's no chance of interference between files written to the current dir by parallel tests

Reply:
Updated to use `t.TempDir` and have also removed the `os.Remove` stuff as it should be handled by the regular cleanup.